### PR TITLE
User instructions

### LINKED ISSUE #6: Report: User's Instructions
User's Instructions explain how to run your project. The instructions should also point out features that we might not otherwise observe by simply experimenting with your application. You may also consider including a "script" that steps us through a sample interaction with the project.

### PR DESCRIPTION
**Pull request by swglad**
Closes #6.

### CODE REVIEW

File: kemeny_sourcemash.tex [hunk 1]
```diff
@@ -43,15 +43,54 @@ \section{Project Overview}
...
+				\item Activate virtual environment: \texttt{.\ flask/bin/activate}
+			\end{enumerate}
+		\end{enumerate}
+	\item Run Locally
+		\begin{enumerate}
+			\item Start up a local server: \texttt{python manage.py server}
+			\item In another terminal session,
+			\begin{enumerate}
+				\item Start up a redis server: \texttt{redis-server \&}
+				\item Start up an instance of the worker: \texttt{python manage.py worker \&}
```
Comment (swglad) [inline kemeny_sourcemash.tex]: Update.

File: kemeny_sourcemash.tex [hunk 2]
```diff
@@ -1,7 +1,7 @@
 \documentclass[11pt]{article}
 \title{\includegraphics[scale=0.65]{solologo} \\~\\ \includegraphics[scale=0.75]{logotext} \\~\\ Kemeny Prize Application 2015}
 \author{
-	Alex Gerstein \\ Dartmouth College '15 \\ Computer Science \\ \texttt{alex.s.gerstein.15@dartmouth.edu} 
+	Alex Gerstein \\ Dartmouth College '15 \\ Computer Science \\ \texttt{alex.s.gerstein.15@dartmouth.edu}
```
Comment (alexgerstein) [inline kemeny_sourcemash.tex]: alexander.s.gerstein.15@dartmouth.edu

File: kemeny_sourcemash.tex [hunk 3]
```diff
@@ -43,15 +43,55 @@ \section{Project Overview}
...
-While 
-
-\subsection{Production: www.sourcemash.com}
-
-When
-
-\subsection{Development: localhost}
-
-When
+Sourcemash is a web application that is hosted online at \url{www.sourcemash.com}. The current development release is available at \url{staging.sourcemash.com}, and the source code for Sourcemash is available at \url{https://github.com/sourcemash/Sourcemash}. Instructions for installation, execution, and testing in the local environment are included in the README and in �2.2 below.
```
Comment (alexgerstein) [inline kemeny_sourcemash.tex]: "Sourcemash is a web application that is hosted online at \url{www.sourcemash.com}" --> "Sourcemash is a web application, located at \url{www.sourcemash.com}"

Comment (alexgerstein) [inline kemeny_sourcemash.tex]: We don't need to give them the staging URL.

File: kemeny_sourcemash.tex [hunk 4]
```diff
@@ -43,15 +43,55 @@ \section{Project Overview}
...
+
+\subsection{Production}
+
+Users are encouraged to use the production release of Sourcemash available at \url{www.sourcemash.com}. The website uses Material Design techniques to create an easy-to-navigate, responsive user experience. Users are encouraged to sign up or log in to take advantages of all of the features that Sourcemash has to offer, and a continuously available help menu is available by pressing the ``?'' button in the bottom right corner of every page.
```
Comment (alexgerstein) [inline kemeny_sourcemash.tex]: "Techniques" --> "Principles"

Comment (alexgerstein) [inline kemeny_sourcemash.tex]: The ? button and sign-in options are clear from visiting the site. I don't think we need to include them in the feature overview. I like how you used the navigation menu to walkthrough the site's core features.

File: kemeny_sourcemash.tex [hunk 5]
```diff
@@ -43,15 +43,55 @@ \section{Project Overview}
...
+
+The navigation menu, pinned to the left side of the page, is the primary tool used to access features and pages on the site, which are delineated below:
+
+\begin{itemize}
+	\item \textbf{Browse} (\url{sourcemash.com/browse}): The Browse page contains an offering of feeds that users can choose to subscribe to. If users do not see a feed that they wish to add, they may add their own using the form at the top of the page.
```
Comment (alexgerstein) [inline kemeny_sourcemash.tex]: Remove "choose to"

File: kemeny_sourcemash.tex [hunk 6]
```diff
@@ -43,15 +43,55 @@ \section{Project Overview}
...
+	\item \textbf{Categorizer} (\url{sourcemash.com/categorizer}): This page offers try-it-yourself access to the Sourcemash categorization engine. The categorizer allows users to input the URL of any online article and returns auto-generated categories based on computational linguistic analysis.
```
Comment (alexgerstein) [inline kemeny_sourcemash.tex]: "try-it-yourself" --> "standalone"

Comment (alexgerstein) [inline kemeny_sourcemash.tex]: "computational linguistic analysis" --> "our categorization algorithm"

File: kemeny_sourcemash.tex [hunk 7]
```diff
@@ -43,15 +43,55 @@ \section{Project Overview}
...
+	\item \textbf{Category or Feed View} (e.g. \url{sourcemash.com/feeds/1}): By clicking on a Category or Feed title under the tabs in the left-pinned navigation menu, users can view all items present in that category or feed and interact with content by up/down-voting and saving items.
```
Comment (alexgerstein) [inline kemeny_sourcemash.tex]: "view all items present in that category or feed and interact with content by up/down-voting and saving items." --> "view and interact with all items present in that category or feed."

File: kemeny_sourcemash.tex [hunk 8]
```diff
@@ -43,15 +43,55 @@ \section{Project Overview}
...
+	\item \textbf{Saved} (\url{sourcemash.com/saved}): Authenticated users can find the list of items that they have bookmarked here.
+	\item \textbf{Profile} (\url{sourcemash.com/profile}): Authenticated users can toggle a ``Suggested Content'' feature here, whereby Sourcemash will display unsubscribed content to a user in order to introduce him or her to recommended, similar content. For example, if users are subscribed to the \emph{New York Times} and are currently viewing a category titled ``Apple Watch,'' they may also be introduced to articles about the Apple Watch from other sources such as TechCrunch, Macrumors, or Bloomberg, if this feature is enabled.
```
Comment (alexgerstein) [inline kemeny_sourcemash.tex]: Authenticated users can manage their settings here. For example, Sourcemash offers a "Show Suggested Content" feature that will display one additional article inside a category page from a feed that a user is not subscribed to. This helps introduce users to new feeds.

File: kemeny_sourcemash.tex [hunk 9]
```diff
@@ -43,15 +43,24 @@ \section{Project Overview}
...
-\subsection{Production: www.sourcemash.com}
+\subsection{Production}
 
-When
+Users are encouraged to use the production release of Sourcemash available at \url{www.sourcemash.com}. The website uses a Material Design framework\cite{Materialize} to create an easy-to-navigate, responsive user experience. The navigation menu, pinned to the left side of the page, is the primary tool used to access the site's core features and pages, which are explained in detail below:
 
-\subsection{Development: localhost}
+\begin{itemize}
+	\item \textbf{Browse} (\url{sourcemash.com/browse}): The Browse page contains an offering of feeds that users can subscribe to. If users do not see a feed that they wish to add, they may add their own using the form at the top of the page.
+	\item \textbf{Categorizer} (\url{sourcemash.com/categorizer}): This page offers standalone access to the Sourcemash categorization engine. The categorizer allows users to input the URL of any online article and returns auto-generated categories based on our categorization algorithm.
```
Comment (alexgerstein) [inline kemeny_sourcemash.tex]: "The categorizer allows users to input the URL of any online article and returns auto-generated categories based on our categorization algorithm." --> "The categorizer allows users to input the URL of any online article and receive auto-generated categories based on our categorization algorithm."

**Pull request merged**